packages/core/src/domain/configuration/configuration.ts

```diff
@@ -149,3 +161,17 @@ export function buildCookieOptions(initConfiguration: InitConfiguration) {
 function mustUseSecureCookie(initConfiguration: InitConfiguration) {
   return !!initConfiguration.useSecureSessionCookie || !!initConfiguration.useCrossSiteSessionCookie
 }
+
+export function serializeConfiguration(configuration: InitConfiguration): Partial<RawTelemetryConfiguration> {
+  return {
+    session_sample_rate: configuration.sampleRate,
+    telemetry_sample_rate: configuration.telemetrySampleRate,
+    telemetry_configuration_sample_rate: configuration.telemetryConfigurationSampleRate,
```

Comment:
Should I update the rum-events-format since telemetryConfigurationSampleRate is ment for tests?

Reply:
Do you mean add this parameter in the configuration telemetry?
If yes, any reason to add it or not?